### ISSUE TITLE
When I use Babel7 to bundle the code I won't be able to access JsSip object directly

### ISSUE THREAD

**Issue by qiulang** [versatica/JsSIP#796]
When I use Babel7 to bundle the code (using webpack) I won't be able to access JsSip object directly, e.g.

```
JsSIP.debug.enable('JsSIP:*');

VM96:1 Uncaught ReferenceError: JsSIP is not defined
    at <anonymous>:1:1

```

I don't know if this is a problem with JsSIP bundle setting, so apologise first if I opened a wrong issue.

**Comment by qiulang** [comment]
Sorry I find it is not because Babel7. 

I remember I can `JsSIP.debug.enable('JsSIP:*')` when I used Babel 6 so I thought that was the reason but now I restore my webpack setting to Babel 6 but I still get "Uncaught ReferenceError: JsSIP is not defined"

I don't what else can go wrong.

**Comment by qiulang** [comment]
Even tryit website can't set that too!

<img width="1147" alt="tryit" src="https://user-images.githubusercontent.com/516243/205891918-c8e6a8b0-f558-4861-93ad-a3b60fd887d6.png">

**Comment by ibc** [comment]
Please use the mailing list for usage questions. This is not a bug in JsSIP.

**Comment by qiulang** [comment]
I did ask it at mailing list

**Comment by qiulang** [comment]
I think I have probably figured out the reason. It is indeed caused by babel. I have found that since 3.12.15 you changed from babel 6 to babel 7 while I have been using babel 6 until recently

Babel 6 uses debug 2.6  while Babel 7 uses 4.x so I have checked it was debug 2.6 in my `node_modules` and for the reason I have not figured out I just don't have debug log then.

When I update my `node_modules` and make sure it has debug 4.x, then I have log again.

It is not a bug in JsSIP but it is definitely an issue for those who still use debug 2.x. So I hope my experience can help them.
